app/src/components/Item/OneItem.js

```diff
@@ -94,7 +111,31 @@ var OneItem = ({ data, clickOnItem ,deleteItem}) => {
 
       <CardActions>
         <div className="button">
-          <Button size="small" >Information</Button>
+          <Button
+            size="small"
+            aria-describedby={id}
+            onClick={handleInfoBtnClick}>
+            Information
+          </Button>
+          <Popover
+            id={id}
+            open={popOpen}
+            anchorEl={anchorEl}
+            onClose={handleInfoBtnClose}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'center',
+            }}
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'center',
+            }}
+          >
+            <Typography className={classes.typography}>
+              {container_name === ' '|| container_name === null  ?  "" : `Contained by: ${container_name}`} <br />
```

Comment:
I understand by the priority, the || is higher than ? :, but can you please wrap the condition part explicitly? 
```suggestion
              {(container_name === ' '|| container_name === null)  ?  "" : `Contained by: ${container_name}`} <br />
```